Logging throws an exception during cleans

```
io.reactivex.rxjava3.exceptions.OnErrorNotImplementedException: The exception was not handled due to missing onError handler in the subscribe() method call. Further reading: https://github.com/ReactiveX/RxJava/wiki/Error-Handling | java.io.FileNotFoundException: /Users/zacsweers/dev/android/personal/CatchUp/build/outputs/logs/last-build-thermals.log (No such file or directory)
        at io.reactivex.rxjava3.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:718)
        at io.reactivex.rxjava3.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:715)
        at io.reactivex.rxjava3.internal.observers.LambdaObserver.onError(LambdaObserver.java:77)
        at io.reactivex.rxjava3.internal.operators.observable.ObservableScanSeed$ScanSeedObserver.onError(ObservableScanSeed.java:118)
        at io.reactivex.rxjava3.internal.operators.observable.ObservableDoOnEach$DoOnEachObserver.onError(ObservableDoOnEach.java:117)
        at io.reactivex.rxjava3.internal.operators.observable.ObservableDoOnEach$DoOnEachObserver.onNext(ObservableDoOnEach.java:97)
        at io.reactivex.rxjava3.internal.operators.observable.ObservableFilter$FilterObserver.onNext(ObservableFilter.java:52)
        at io.reactivex.rxjava3.internal.operators.observable.ObservableMap$MapObserver.onNext(ObservableMap.java:63)
        at io.reactivex.rxjava3.internal.operators.observable.ObservableInterval$IntervalObserver.run(ObservableInterval.java:82)
        at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectPeriodicTask.run(ScheduledDirectPeriodicTask.java:35)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
        at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.io.FileNotFoundException: /Users/zacsweers/dev/android/personal/CatchUp/build/outputs/logs/last-build-thermals.log (No such file or directory)
        at java.base/java.io.FileOutputStream.open0(Native Method)
        at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
        at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
        at kotlin.io.FilesKt__FileReadWriteKt.appendBytes(FileReadWrite.kt:115)
        at kotlin.io.FilesKt__FileReadWriteKt.appendText(FileReadWrite.kt:142)
        at kotlin.io.FilesKt__FileReadWriteKt.appendText$default(FileReadWrite.kt:142)
        at slack.gradle.util.AppleSiliconThermals$start$4.accept(ThermalsWatcher.kt:297)
        at slack.gradle.util.AppleSiliconThermals$start$4.accept(ThermalsWatcher.kt:266)
        at io.reactivex.rxjava3.internal.operators.observable.ObservableDoOnEach$DoOnEachObserver.onNext(ObservableDoOnEach.java:93)
        ... 10 more
Exception in thread "RxComputationThreadPool-6" io.reactivex.rxjava3.exceptions.OnErrorNotImplementedException: The exception was not handled due to missing onError handler in the subscribe() method call. Further reading: https://github.com/ReactiveX/RxJava/wiki/Error-Handling | java.io.FileNotFoundException: /Users/zacsweers/dev/android/personal/CatchUp/build/outputs/logs/last-build-thermals.log (No such file or directory)
        at io.reactivex.rxjava3.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:718)
        at io.reactivex.rxjava3.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:715)
        at io.reactivex.rxjava3.internal.observers.LambdaObserver.onError(LambdaObserver.java:77)
        at io.reactivex.rxjava3.internal.operators.observable.ObservableScanSeed$ScanSeedObserver.onError(ObservableScanSeed.java:118)
        at io.reactivex.rxjava3.internal.operators.observable.ObservableDoOnEach$DoOnEachObserver.onError(ObservableDoOnEach.java:117)
        at io.reactivex.rxjava3.internal.operators.observable.ObservableDoOnEach$DoOnEachObserver.onNext(ObservableDoOnEach.java:97)
        at io.reactivex.rxjava3.internal.operators.observable.ObservableFilter$FilterObserver.onNext(ObservableFilter.java:52)
        at io.reactivex.rxjava3.internal.operators.observable.ObservableMap$MapObserver.onNext(ObservableMap.java:63)
        at io.reactivex.rxjava3.internal.operators.observable.ObservableInterval$IntervalObserver.run(ObservableInterval.java:82)
        at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectPeriodicTask.run(ScheduledDirectPeriodicTask.java:35)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
        at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.io.FileNotFoundException: /Users/zacsweers/dev/android/personal/CatchUp/build/outputs/logs/last-build-thermals.log (No such file or directory)
        at java.base/java.io.FileOutputStream.open0(Native Method)
        at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
        at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
        at kotlin.io.FilesKt__FileReadWriteKt.appendBytes(FileReadWrite.kt:115)
        at kotlin.io.FilesKt__FileReadWriteKt.appendText(FileReadWrite.kt:142)
        at kotlin.io.FilesKt__FileReadWriteKt.appendText$default(FileReadWrite.kt:142)
        at slack.gradle.util.AppleSiliconThermals$start$4.accept(ThermalsWatcher.kt:297)
        at slack.gradle.util.AppleSiliconThermals$start$4.accept(ThermalsWatcher.kt:266)
        at io.reactivex.rxjava3.internal.operators.observable.ObservableDoOnEach$DoOnEachObserver.onNext(ObservableDoOnEach.java:93)
        ... 10 more

```

--- a/slack-plugin/src/main/kotlin/slack/gradle/util/ThermalsWatcher.kt
+++ b/slack-plugin/src/main/kotlin/slack/gradle/util/ThermalsWatcher.kt
@@ -19,6 +19,7 @@ import com.sun.jna.Library
 import com.sun.jna.Native
 import io.reactivex.rxjava3.core.Observable
 import io.reactivex.rxjava3.disposables.Disposable
+import io.reactivex.rxjava3.observers.DisposableObserver
 import io.reactivex.rxjava3.subjects.BehaviorSubject
 import java.io.File
 import java.time.Duration
@@ -301,7 +302,21 @@ internal class AppleSiliconThermals(
             is ThermalsData -> ThermalsData(acc.logs + next)
           }
         }
-        .subscribe { emissions.onNext(it) }
+        .subscribeWith(
+          object : DisposableObserver<Thermals>() {
+            override fun onNext(thermals: Thermals) {
+              emissions.onNext(thermals)
+            }
+
+            override fun onError(e: Throwable) {
+              System.err.println("Error in thermals watcher:\n${e.stackTraceToString()}")
+            }
+
+            override fun onComplete() {
+              // Nothing to do
+            }
+          }
+        )
   }
 
   override fun peek(): Thermals {

